### PR TITLE
Image Builder Naming Conventions for _Splatmaps

### PR DESCRIPTION
**Pull request by HogJonny-AMZN**
## What does this PR do?
Adds naming convention support for _linear and _splatmaps to the image builder profiles.

## How was this PR tested?
Locally created images using conventions, cooked and then utilized in the engine.

### CODE REVIEW

File: Gems/Atom/Asset/ImageProcessingAtom/Assets/Config/ImageBuilder.settings [hunk 1]
```diff
@@ -201,6 +201,11 @@
             "_specular": [ "Reflectance" ],
 			"_SpecularReflectance": [ "Reflectance" ],
 			"_thickness": [ "Reflectance" ],
+			"_linear": [ "Reflectance" ],
+			"_sm": [ "Reflectance" ],
+			"_splat": [ "Reflectance" ],
+			"_splatmap": [ "Reflectance" ],
```
Comment (cgalvan) [inline Gems/Atom/Asset/ImageProcessingAtom/Assets/Config/ImageBuilder.settings:207]: Nit: Looks like some extra tab spacing here

File: Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Registry/AssetProcessorGemConfig.setreg [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "O3DE": {
+        "Atom": {
+            "RPI": {
+                "MaterialTypeBuilder": {
+                    "CreateAllPropertiesMaterial": true
+				}
```
Comment (cgalvan) [inline Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Registry/AssetProcessorGemConfig.setreg:7]: Nit: Fix extra tab spacing on this `}`

File: Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/__init__.py [hunk 3]
```diff
@@ -8,17 +8,15 @@
 #
 #
 # -------------------------------------------------------------------------
-"""! @brief
-DCCsi/Tools/DCC/Maya/__init__.py
+"""! TThis init allows us to treat Maya as a DCCsi tools python package
```
Comment (cgalvan) [inline Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/__init__.py:11]: Nit:
```suggestion
"""! This init allows us to treat Maya as a DCCsi tools python package
```

**Review by cgalvan — APPROVED** [review]

**Comment by HogJonny-AMZN** [comment]
replaced with: https://github.com/o3de/o3de/pull/12470

**Pull request closed without merging**